### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by maf-embraceit**

### CODE REVIEW

**Review by BushraAsif — COMMENTED** [review]
Changes reviewed

**Review by BushraAsif — APPROVED** [review]

**Review by BushraAsif — COMMENTED** [review]
Changes reviewed

File: tests/test_utils.py [hunk 1]
```diff
@@ -1,130 +1,130 @@
-from __future__ import absolute_import, unicode_literals
```
Comment (emicha) [inline tests/test_utils.py:1]: You might need to update your IDE. There is no real changes in this file:
image

File: tests/test_resource.py [hunk 2]
```diff
@@ -1,92 +1,92 @@
-from __future__ import absolute_import, unicode_literals
```
Comment (emicha) [inline tests/test_resource.py:1]: You might need to check your IDE. There is no real changes in this file

File: tests/test_utils.py (hunk 1 above)
Comment (maf-embraceit) [reply tests/test_utils.py]: Yes, this is an IDE/OS issue. I've reverted the file.

File: tests/test_resource.py (hunk 2 above)
Comment (maf-embraceit) [reply tests/test_resource.py]: Yes this is an IDE/OS issue. I've reverted the file.

**Review by emicha — APPROVED** [review]

**Pull request merged**